extra-resources/mods/Mythos/chinese.cardset

```diff
@@ -170,9 +155,8 @@ card('GUAN YU') {
         summon 2 of 'Terracotta Soldier' to 'you' zone 'Hand'
     }
 }
-// Info: http://en.wikipedia.org/wiki/Eight_Immortals
-// Image: http://upload.wikimedia.org/wikipedia/commons/9/9a/Eight_Immortals_Crossing_the_Sea_-_Project_Gutenberg_eText_15250.jpg
-// License: Public Domain
+// https://en.wikipedia.org/wiki/Eight_Immortals
+// https://commons.wikimedia.org/wiki/File:Eight_Immortals_Crossing_the_Sea_-_Project_Gutenberg_eText_15250.jpg
```

Comment:
I think we'll just have to leave this image out then, and revert back to the default image. With some luck maybe we'll find another one at some point, that's _truly_ public domain (or CC).